archivers/libzzip/Portfile

```diff
@@ -18,26 +18,22 @@ long_description    The ZZIPlib provides read access on ZIP-archives. The \
                     access files being either real files or zipped files, \
                     both with the same filepath.
 
-checksums           rmd160  d401fc327138d5b997c0cd318f808f95cea1f937 \
-                    sha256  c5cf1c11a6019862e5b6daea4d7c72124affe687b9637effc7ad3caf2b7b1d96 \
-                    size    1132209
+checksums           rmd160  eee9554167b1d27a0973ccee191df811c6126f43 \
+                    sha256  3df044caee3cdeaea3384fe67a452430d92e874dfb5c041c35c1339268025b07 \
+                    size    1150828
 
 depends_build       port:pkgconfig \
-                    port:python27
+                    port:python38 \
+                    port:cmake
```

Comment:
Don't manually add the `port:cmake` dependency; the cmake portgroup does it for you. But do use `depends_build-append` instead of `depends_build` so that you don't overwrite the portgroup's dependencies.